mayastor/src/bdev/nexus/nexus_child.rs

```diff
@@ -168,7 +154,7 @@ impl Display for NexusChild {
                 "{}: {:?}/{:?}, blk_cnt: {}, blk_size: {}",
                 self.name,
                 self.state,
-                self.status(),
+                self.state(),
```

Comment:
Looks like you can remove one of these, also on line 167 below.

Reply:
Thanks, made the changes.